tests/hf_hydrodata/test_data_catalog.py

```diff
@@ -149,3 +149,29 @@ def test_dataset_version_default():
     )
     assert row["id"] == "167"
     assert row["dataset_version"] == "0.9"
+
+
+def test_get_citations_usgs():
+    """Test for get_citations function with return DataFrame."""
+    citation = hf.get_citations(dataset="usgs_nwis")
+
+    assert isinstance(citation, str)
+
+
+def test_get_citations_ameriflux():
+    """Test for get_citations function with return DataFrame."""
+    citation = hf.get_citations(dataset="ameriflux")
+
+    assert isinstance(citation, str)
+
+
+def test_get_citations_jasechko():
+    """Test citations for jasechko_2024 dataset."""
+    t = hf.get_citations("jasechko_2024")
+    assert "10.1038/s41586-023-06879-8" in t
+
+
+def test_get_citations_fan():
```

Comment:
Maybe add a test that asserts on the exception of bad inputs

Reply:
Good idea! I am adding one in.